rocketpy/motors/fluid.py

```diff
@@ -60,3 +60,10 @@ def __str__(self):
         """
 
         return f"Fluid: {self.name}"
+
+    def to_dict(self, _):
```

Comment:
I usually see this behavior in other packages, what do you think about it? I dont really understand the reasons of that tho.

```suggestion
    def to_dict(self, *):
```

Reply:
Well, actually I think I will simply revert to having the parameter `include_outputs` in all of them, and disable the warning if the method does not use it. That brings better standardization, since the call `to_dict(include_outputs=...)` would always work.

The `**` would be a better option than I used here, since it allows for the same interface as I mentioned. However, it also allows for other parameter names, and changing it in the future would be a breaking change.